src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const splitedString = sourceString.split(';');
```

Comment:
this constant actually contains an array, so, I think there is should be a more appropriate constant name.